Symfony/CS/Fixer/PSR2/EofEndingFixer.php

```diff
@@ -25,13 +26,25 @@ public function fix(\SplFileInfo $file, $content)
     {
         // [Structure] A file must always end with a linefeed character
 
-        $content = rtrim($content);
+        if (empty($content)) {
+            return $content;
+        }
+
+        $tokens = Tokens::fromCode($content);
+        $count = $tokens->count();
+        if (0 === $count) {
+            return;
```

Comment:
good thing it is ;)  
(you did refer to https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.9/Symfony/CS/Tests/Fixer/PSR2/EofEndingFixerTest.php#L62 I hope ;) )